catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -810,6 +874,41 @@ public CreateResponse create(CreateRequest createRequest)
 
         Exception ingestError = null;
         try {
+            if (policy != null) {
+                Map<String, Serializable> unmodifiablePropertiesMap = Collections.unmodifiableMap(
+                        createReq.getProperties());
+                HashMap<String, Set<String>> requestPolicyMap = new HashMap<>();
+                for (Metacard metacard : createReq.getMetacards()) {
+                    HashMap<String, Set<String>> itemPolicyMap = new HashMap<>();
+                    for (PolicyPlugin plugin : policy) {
+                        try {
+                            PolicyResponse policyResponse = plugin.processPreCreate(metacard,
+                                    unmodifiablePropertiesMap);
+                            buildPolicyMap(itemPolicyMap, policyResponse.itemPolicy().entrySet());
+                            buildPolicyMap(requestPolicyMap, policyResponse.operationPolicy().entrySet());
+                        } catch (PluginExecutionException e) {
+                            LOGGER.info(
+                                    "Plugin processing failed. This is allowable. Skipping to next plugin.",
+                                    e);
+                        }
+                    }
+                    metacard.setAttribute(new AttributeImpl(Metacard.SECURITY, itemPolicyMap));
+                }
+                createReq.getProperties().put(PolicyPlugin.OPERATION_SECURITY, requestPolicyMap);
+            }
+
+            if (access != null) {
+                for (AccessPlugin plugin : access) {
+                    try {
+                        plugin.processPreCreate(createReq);
+                    } catch (PluginExecutionException e) {
```

Comment:
StopProcessingException is the one that we would normally stop the process for. It is a good question though, does having a PluginExecutionException even make sense for an access plugin? Maybe we only have StopProcessingException?